_posts/en/meetings/2016-05-19-meeting.md

```diff
@@ -16,22 +16,24 @@ version: 1
  
 ## Main topics
  
-- Versionbits GetBlockTemplate
+- Versionbits GetBlockTemplate and CSV activation
 
 ## Notes / short topics
 
 - This was a short meeting with low attendance because a lot of developers where traveling to the [coredev hacking event](http://coredev.tech/).
 - Luke-jr wonders if someone has looked at the latest OpenSSL exploits. When releasing 0.12.2 it might makes sense to bump OpenSSL with it.
 
-## Versionbits GetBlockTemplate
+## Versionbits GetBlockTemplate and CSV activation
  
 ### background
  
 Getblocktemplate is the new decentralized Bitcoin mining protocol, openly developed by the Bitcoin community over mid 2012. It supersedes the old getwork mining protocol. ([Wiki](https://en.bitcoin.it/wiki/Getblocktemplate)-link)
+
+CheckSequenceVerify or [BIP112][] is one of the time-lock related BIPs released in 0.12.1 which is the first softfork that should activate with [BIP9][] versionbits.
  
 ### meeting comments
  
-It is fairly important to get it released before [BIP112][] CSV activates. There are some applications that want CSV and it would be nice to not have both CSV and segwit activate at the same time, so while not critical it would be nice if it could be activated within a decent timeframe. backports to 0.11 or 0.12 CPFP testing/review would get the Eligius mining pool on board with CSV.
```

Comment:
This change is wrong. GBT VB is not related closely to Eligius at all, but needed in general.